Devices/igtlioVideoDevice.h

```diff
@@ -88,9 +81,11 @@ class OPENIGTLINKIO_DEVICES_EXPORT VideoDevice : public Device
   
   GenericDecoder* VideoStreamDecoderH264;
   
+  std::map<std::string,GenericDecoder*> decoders;
+  
   SourcePicture* pDecodedPic;
   
-  CodecTypes currentCodecType;
+  std::string currentCodecType;
```

Comment:
There is inconsistent variable naming here. Could you refactor all of the member variables to the same convention?